src/components/learner-credit-management/OfferUtilizationAlerts.jsx

```diff
@@ -0,0 +1,108 @@
+import React, { useEffect, useState } from 'react';
+import PropTypes from 'prop-types';
+import { Alert } from '@edx/paragon';
+import { Info, WarningFilled } from '@edx/paragon/icons';
+import { sendEnterpriseTrackEvent } from '@edx/frontend-enterprise-utils';
+
+import ContactCustomerSupportButton from '../ContactCustomerSupportButton';
+import {
+  LOW_REMAINING_BALANCE_PERCENT_THRESHOLD,
+  NO_BALANCE_REMAINING_DOLLAR_THRESHOLD,
+} from './data/constants';
+
+const OfferUtilizationAlerts = ({
+  className,
+  percentUtilized,
+  remainingFunds,
+  enterpriseUUID,
+}) => {
+  const [isWarningAlertShown, setIsWarningAlertShown] = useState(false);
+
+  useEffect(() => {
+    const isWarningTheshold = (
+      percentUtilized > LOW_REMAINING_BALANCE_PERCENT_THRESHOLD
```

Comment:
`>= `?

Reply:
Hmm, based on the current messaging in the `Alert` (i.e., "... is **over 75.0%** utilized), I think this should be kept as `>`?